docs/concepts/save-query.asciidoc

```diff
@@ -1,39 +1,40 @@
 [[save-load-delete-query]]
-== Save a query
-A saved query is a collection of query text and filters that you can
-reuse in any app with a query bar, like <<discover, *Discover*>> and <<dashboard, *Dashboard*>>. Save a query when you want to:
+=== Save a query
+A saved query is a portable collection of query text and filters that you can reuse in
+<<discover, Discover>> and <<dashboard, Dashboard>>. Save a query when you want to:
```

Comment:
What I wrote earlier is correct: you can reuse in multiple apps, and Discover/Dashboard are examples. You can remove the examples if you want, but I think it would be misleading as written here.